client/js/verification-portal/pregnancy-resource-center/form.js

```diff
@@ -260,37 +261,37 @@ class VerificationPortalForm extends Component {
 					<Heading text="HOURS" />
 					<Spacer height="50px" />
 					<label>Sunday Hours</label>
-					<Field name="hours[0].open" component={Input} type="time" />
-					<Field name="hours[0].close" component={Input} type="time" />
+					<Field name="hours[0].open" component={Time} placeholder={'Open'} />
+					<Field name="hours[0].close" component={Time} placeholder={'Close'} />
 
 					<label>Monday Hours</label>
-					<Field name="hours[1].open" component={Input} type="time" />
-					<Field name="hours[1].close" component={Input} type="time" />
+					<Field name="hours[1].open" component={Time} placeholder={'Open'} />
+					<Field name="hours[1].close" component={Time} placeholder={'Close'} />
 
 					<label>Tuesday Hours</label>
-					<Field name="hours[2].open" component={Input} type="time" />
-					<Field name="hours[2].close" component={Input} type="time" />
+					<Field name="hours[2].open" component={Time} placeholder={'Open'} />
+					<Field name="hours[2].close" component={Time} placeholder={'Close'} />
 
 					<label>Wednesday Hours</label>
-					<Field name="hours[3].open" component={Input} type="time" />
-					<Field name="hours[3].close" component={Input} type="time" />
+					<Field name="hours[3].open" component={Time} placeholder={'Open'} />
+					<Field name="hours[3].close" component={Time} placeholder={'Close'} />
 
 					<label>Thursday Hours</label>
-					<Field name="hours[4].open" component={Input} type="time" />
-					<Field name="hours[4].close" component={Input} type="time" />
+					<Field name="hours[4].open" component={Time} placeholder={'Open'} />
+					<Field name="hours[4].close" component={Time} placeholder={'Close'} />
 
 					<label>Friday Hours</label>
-					<Field name="hours[5].open" component={Input} type="time" />
-					<Field name="hours[5].close" component={Input} type="time" />
+					<Field name="hours[5].open" component={Time} placeholder={'Open'} />
+					<Field name="hours[5].close" component={Time} placeholder={'Close'} />
 
 					<label>Saturday Hours</label>
-					<Field name="hours[6].open" component={Input} type="time" />
-					<Field name="hours[6].close" component={Input} type="time" />
+					<Field name="hours[6].open" component={Time} placeholder={'Open'} />
+					<Field name="hours[6].close" component={Time} placeholder={'Close'} />
 				</div>
 
 				<div className={classes.parent}>
 					<Field
-						name="verifiedData.hours.verified"
+						name="verifiedTimeData.hours.verified"
```

Comment:
@bhavyaab What is the reasoning for this change? All of the values that need to be verified are stored in the `verifiedData` object.

Reply:
It was by mistake.